src/couchConfig.js

```diff
@@ -0,0 +1,371 @@
+/*
+Copyright 2017 OCAD University
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/fluid-couch-config/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion");
+fluid.setLogging(true);
```

Comment:
I don't think that we should include this `fluid.setLogging(true)` here. I think it should be under control of the code that is using this file. Which in our case would only need to be:

- `examples/js/couchConfigBasic.js`

As I think the test-running machinery enables logging.